includes/class-engagement-cloud-activator.php

```diff
@@ -60,21 +60,21 @@ public function __construct( $plugin_name, $callback_url ) {
 	/**
 	 * Executed upon plugin activation.
 	 *
-	 * Executed upon plugin activation and posts to dotmailer's
+	 * Executed upon plugin activation and posts to Engagement Cloud
 	 * tracking site to notify that the plugin has been activated.
 	 *
 	 * @since    1.0.0
 	 */
 	public function activate() {
 		global $wpdb;
-		$dotmailer_table_name = $wpdb->prefix . $this->plugin_name;
+		$engagement_cloud_table_name = $wpdb->prefix . $this->plugin_name;
```

Comment:
I was gonna mention that on Monday scrumm. Since that table is build based on plugin name I think we have 3 options here:
- Users reconnecting rather than reinstalling (although reinstalling might be also needed if we change the repo name).
- Changing the code to hardcode the old table name
  - Another version could be, instead of hardcoding the old table name hardcode a migration script which moves the plugin id from the old table to the new one. 
- Keep the current plugin name (that is dotmailer_email_marketing).